src/foam/nanos/theme/Theme.js

```diff
@@ -102,157 +112,244 @@ foam.CLASS({
-      section: 'colours'
+      section: 'colours',
+      value: 'unset'
     },
     {
       class: 'Color',
       name: 'approval1',
-      section: 'colours'
+      section: 'colours',
+      value: 'unset'
     },
     {
       class: 'Color',
       name: 'approval2',
-      section: 'colours'
+      section: 'colours',
+      value: 'unset'
     },
     {
       class: 'Color',
       name: 'approval3',
-      section: 'colours'
+      section: 'colours',
+      value: 'unset'
     },
     {
       class: 'Color',
       name: 'approval4',
-      section: 'colours'
+      section: 'colours',
+      value: 'unset'
     },
     {
       class: 'Color',
       name: 'approval5',
-      section: 'colours'
+      section: 'colours',
+      value: 'unset'
     },
     {
       class: 'Color',
       name: 'warning1',
-      section: 'colours'
+      section: 'colours',
+      value: 'unset'
     },
     {
       class: 'Color',
       name: 'warning2',
-      section: 'colours'
+      section: 'colours',
+      value: 'unset'
     },
     {
       class: 'Color',
       name: 'warning3',
-      section: 'colours'
+      section: 'colours',
+      value: 'unset'
     },
     {
       class: 'Color',
       name: 'warning4',
-      section: 'colours'
+      section: 'colours',
+      value: 'unset'
     },
     {
       class: 'Color',
       name: 'warning5',
-      section: 'colours'
+      section: 'colours',
+      value: 'unset'
     },
     {
       class: 'Color',
       name: 'destructive1',
-      section: 'colours'
+      section: 'colours',
+      value: 'unset'
     },
     {
       class: 'Color',
       name: 'destructive2',
-      section: 'colours'
+      section: 'colours',
+      value: 'unset'
     },
     {
       class: 'Color',
       name: 'destructive3',
-      section: 'colours'
+      section: 'colours',
+      value: 'unset'
     },
     {
       class: 'Color',
       name: 'destructive4',
-      section: 'colours'
+      section: 'colours',
+      value: 'unset'
     },
     {
       class: 'Color',
       name: 'destructive5',
-      section: 'colours'
+      section: 'colours',
+      value: 'unset'
     },
     {
       class: 'Color',
       name: 'grey1',
-      section: 'colours'
+      section: 'colours',
+      value: 'unset'
     },
     {
       class: 'Color',
       name: 'grey2',
-      section: 'colours'
+      section: 'colours',
+      value: 'unset'
     },
     {
       class: 'Color',
       name: 'grey3',
-      section: 'colours'
+      section: 'colours',
+      value: 'unset'
     },
     {
       class: 'Color',
       name: 'grey4',
-      section: 'colours'
+      section: 'colours',
+      value: 'unset'
     },
     {
       class: 'Color',
       name: 'grey5',
-      section: 'colours'
+      section: 'colours',
+      value: 'unset'
     },
     {
       class: 'Color',
       name: 'black',
-      section: 'colours'
+      section: 'colours',
+      value: 'black'
```

Comment:
Wouldn't this make sense as `unset` as well?

Reply:
Maybe? This is mostly a selfish default but I feel like it isn't a totally unreasonable default. black being black doesn't feel particularly unexpected lol. I can switch this to unset for consistency if you think that'd be better.

Reply:
`unset` is more consistent with the other properties, and it should default to `black` in most cases anyway, shouldn't it? Isn't the default `color` on most browsers black? If whoever's using it has a different colour in a parent element we'd want this property's behaviour to be consistent with the other properties, right?

Reply:
Yup that make sense. I'll make this consistent with the rest :)